cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -475,6 +475,7 @@ func main() {
 			factoryForNamespace.Storage().V1beta1().CSIStorageCapacities(),
 			*capacityPollInterval,
 			*capacityImmediateBinding,
+			*operationTimeout,
```

Comment:
Setting some timeout makes sense, I just wonder whether we should use `operationTimeout` for it. It's currently defined as
```
        operationTimeout     = flag.Duration("timeout", 10*time.Second, "Timeout for waiting for creation or deletion of a volume")                                                                                                                                                                                                                                                                    
```

Perhaps change that into `Timeout for volume operations (creation, deletion. capacity queries)`?

Reply:
Creation and deletion all use same timeoutOperation. I think we should keep this pattern. We don't need define too much timeout which is unnecessary

Reply:
I agree, but then the description should be changed to include the new usage. Querying capacity is not "creation or deletion of a volume" (current description).

Reply:
Oh！I miss it, I will add it

Reply:
@pohly done~